_meta/kibana/5.x/index-pattern/mysqlbeat.json

```diff
@@ -0,0 +1,6 @@
+  "fieldFormatMap": "{\"@timestamp\": {\"id\": \"date\"}}", 
+  "timeFieldName": "@timestamp", 
+  "title": "mysqlbeat-*"
+}
```

Comment:
add newline at end of file (this applies to a few other files in this PR)

Reply:
will be addressed in new PR